app/lib/common/widgets/invite_result_card.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+import 'package:acter/features/home/widgets/user_avatar.dart';
+
+class InviteResultCard extends StatelessWidget {
+  final String? displayName;
+  final String? username;
+
+  const InviteResultCard({
+    super.key,  this.displayName, required this.username,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        const UserAvatarWidget(),
+        Container(
+          margin: const EdgeInsets.only(left: 10),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children:  [
+              Text(displayName!),
```

Comment:
Does your linter not complain because of not setting ```const``` to some widgets?

Reply:
It does in some , in this file I have no linter complains .